test/int/cluster_test.go

```diff
@@ -112,15 +112,66 @@ var _ = Describe("AtlasCluster", func() {
 		})
 	}
 
-	performUpdate := func() {
+	performUpdate := func(timeout interface{}) {
 		Expect(k8sClient.Update(context.Background(), createdCluster)).To(Succeed())
 
 		Eventually(testutil.WaitFor(k8sClient, createdCluster, status.TrueCondition(status.ReadyType), validateClusterUpdatingFunc()),
-			1200, interval).Should(BeTrue())
+			timeout, interval).Should(BeTrue())
 
 		lastGeneration++
 	}
 
+	Describe("Create cluster & change ReplicationSpecs", func() {
```

Comment:
What do you think about adding another a bit more complicated use case and specifying two different regions for one replicationSpec (and some configuration for analytics/read-only nodes)?

The current use-case is nice as it covers our merging logic (take the things from Atlas and merge ours on top of the default ones) though we don't have the coverage for more complicated scenario when the user spreads his cluster over regions)